python/ray/_private/parameter.py

```diff
@@ -68,6 +68,8 @@ class RayParams:
             be created.
         worker_path (str): The path of the source code that will be run by the
             worker.
+        setup_worker_path (str): The path of the source code that will be run
```

Comment:
FYI @iycheng we should be able to leverage this for pluggability in setting up runtime_env